questionReply/questionReply-war/src/main/java/org/silverpeas/components/questionreply/control/QuestionReplySessionController.java

```diff
@@ -539,45 +533,17 @@ public ContainerPositionInterface getSilverContentIdPosition() throws QuestionRe
   }
 
   public String genericWriters() throws QuestionReplyException {
-    GenericPanel gp = new GenericPanel();
-    String webContext = URLUtil.getApplicationURL();
-    String theURL = webContext + "/RquestionReply/" + getComponentId() + "/EffectiveRelaunch";
-    String cancelURL =
-        webContext + "/RquestionReply/" + getComponentId() + "/ConsultQuestionQuery?questionId=" +
-            getCurrentQuestion().getPK().getId();
-    Pair<String, String> hostComponentName = new Pair<>(getComponentLabel(),
-        webContext + "/RquestionReply/" + getComponentId() + "/Main");
-    Pair<String, String> hostPath1 = new Pair<>(getCurrentQuestion().getTitle(),
-        "/RquestionReply/" + getComponentId() + "/ConsultQuestionQuery?questionId=" +
-            getCurrentQuestion().getPK().getId());
-    Pair<String, String>[] hostPath = new Pair[]{hostPath1};
-
-    gp.resetAll();
-
-    gp.setHostSpaceName(getSpaceLabel());
-    gp.setHostComponentName(hostComponentName);
-    gp.setHostPath(hostPath);
-
-    gp.setCancelURL(cancelURL);
-
-    gp.setGoBackURL(theURL);
-
-    gp.setPanelProvider(new ExpertPanel(getLanguage(), getCurrentQuestionWriters()));
-
-    gp.setPopupMode(false);
-    gp.setMultiSelect(true);
-    gp.setSelectable(true);
-    setGenericPanel("QR", gp);
-
-    return GenericPanel.getGenericPanelURL("QR");
+    // This method is no more used at the moment (Silverpeas 6.0)
+    return "";
   }
 
   /*
-   * Relance et modifie la question courante
+   * @deprecated (This method is no more used at the moment - Silverpeas 6.0. If necessary,
+   * implementation must be completed to get recipients)
    */
+  @Deprecated
   public void relaunchRecipients() throws QuestionReplyException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)